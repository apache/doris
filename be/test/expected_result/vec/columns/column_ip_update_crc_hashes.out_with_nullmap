column: IPv4 with hashes: [0,2033013095,207242731,1702897183,3034981006,2033013095,1702897183,0,2033013095,1682066472,1702897183,2033013095,3034981006,1702897183,0]
column: IPv6 with hashes: [0,181178161,0,118872628,0]
