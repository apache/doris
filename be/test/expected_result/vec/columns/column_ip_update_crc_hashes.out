column: IPv4 with hashes: [1702897183,2033013095,207242731,1702897183,3034981006,2033013095,1702897183,207242731,2033013095,1682066472,1702897183,2033013095,3034981006,1702897183,2033013095]
column: IPv6 with hashes: [2964864619,181178161,1479828048,118872628,592813460]
